lib/OpenQA/Utils.pm

```diff
@@ -305,15 +306,25 @@ my %bugrefs = (
 );
 my %bugurls = reverse %bugrefs;
 
+sub bugref_regex {
+    my $regex = join('|', keys %bugrefs);
+    return qr/(($regex)#(\d+))(?![\w\d]+)/;
```

Comment:
You're right. These were just some leftovers from experimenting with the expression. I fixed it and also the occurrence in `href_to_bugref`.